libs/ui/theme/src/theme.ts

```diff
@@ -12,5 +13,23 @@ export const theme = extendTheme({
         },
       },
     },
+    Button: {
```

Comment:
I like this part, good that we apply one style to all the buttons, I think we should use themes more often to keep consistency.